Permission and Access Helper/force-app/main/default/classes/FieldPermissionsSelector.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>57.0</apiVersion>
```

Comment:
Sync Api version for all the files. There are some files with 56 apiVersion and some files with 57 apiVersion

Reply:
Done